lib/mi_bridges/driver/job_income_more_about_page.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class JobIncomeMoreAboutPage < BasePage
+      TITLE = /More About .*'s Job/
+
+      delegate :members, to: :snap_application
+
+      def setup; end
+
+      def fill_in_required_fields
+        fill_in "Name of Employer", with: current_member.employed_employer_name
+        select current_member.employed_pay_interval,
+          from: "How often does Complete (40) get paid? "\
+                "This is Complete (40)'s pay period"
+
+        if hourly?
+          fill_in_hourly
+        else
+          fill_in_salary
```

Comment:
if someone is not hourly, that doesn't mean that they have a salary. The interval options we provide are: 

```
["Hourly", "Daily", "Weekly", "2 weeks", "Monthly", "Yearly"],
```
I'd imagine that many hourly workers are paid every 2 weeks, for example.

I wonder if this is a case where we need MichiganBenefits.org to map more closely to MI Bridges (eg: a question that asks whether someone is paid hourly or with a salary)... cc @alanjosephwilliams

Reply:
I took the info directly from MI Bridges and implemented it in the context of MB.org so we're doing a more close 1:1 match in this area.